<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicImageViewControl</class>
 <widget class="QWidget" name="BasicImageViewControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="loadButton">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
